### ISSUE TITLE
How do you use the knob?

### ISSUE THREAD

**Issue by SBoudrias** [google/yapf#213]
The documentation is very vague about this and doesn't gives any hint as to how you'd use them.

**Comment by gwelymernans** [comment]
Sorry, I'll change that. In the meantime, you can set up a `.style.yapf` in the base directory of your project. In it, you will have something like:

```
[style]
based_on_style = pep8
spaces_before_comment = 3
```

Each knob is listed with a description if you use `yapf --style-help`.

**Comment by SBoudrias** [comment]
Oh also, yapf expect specific ini section depending in which file we store it.

- `[style]` for `.style.yapf`
- `[yapf]` for `config.cfg`

**Comment by EKenny31** [comment]
The documentation is still very vague.

**Comment by vinnichase** [comment]
Yes i also stumbled over it. can you please clarify if the capital KNOBs can just be used with lower case letters?

**Comment by chashiv** [comment]
> Oh also, yapf expect specific ini section depending in which file we store it.
> 
> * `[style]` for `.style.yapf`
> * `[yapf]` for `config.cfg`

If I am not wrong, shouldn't `[yapf]` for `setup.cfg` ?
Thanks in advance :)

**Comment by chashiv** [comment]
> Oh also, yapf expect specific ini section depending in which file we store it.
> 
> * `[style]` for `.style.yapf`
> * `[yapf]` for `config.cfg`

If I am not wrong should not `[yapf]` for `setup.cfg` ?
Thanks in advance :)

**Comment by chashiv** [comment]
> Yes i also stumbled over it. can you please clarify if the capital KNOBs can just be used with lower case letters?

@vinnichase Yes, they can be :P
